### ISSUE TITLE
[zsh completion] missing _zpm

### ISSUE THREAD

**Issue by fennecdjay** [zpm-project/zpm-zsh#24]

**Comment by desyncr** [comment]
@fennecdjay You said you have a very basic _zpm completion, is that the case? What do you think about including it into the code base?

**Comment by fennecdjay** [comment]
It still breaks file completion for now, and I do not manage to have completion canditates working for remove/disable. 😞 
I'll have a further look at it, and probably start by sharing a gist.

**Comment by fennecdjay** [comment]
Obviously far from perfect, but [this](https://gist.github.com/fennecdjay/c0721f406eb9d83fa9dc2f42d9ca482f) is what I came up with.

**Comment by fennecdjay** [comment]
Did you try it?

**Comment by desyncr** [comment]
Yep I tried it. It's a good starting point but it needs some work. I'm gonna work on it later tonight.

**Comment by fennecdjay** [comment]
Good luck 😆
